roles/explorer.nix

```diff
@@ -130,30 +129,12 @@ in {
     SupplementaryGroups = "cardano-node";
   };
 
-  services.cardano-explorer-api = {
-    enable = true;
-    port = 8100;
-    package = cardano-rest-pkgs.cardanoRestHaskellPackages.cardano-explorer-api.components.exes.cardano-explorer-api;
-  };
-
-  systemd.services.cardano-explorer-api = {
-    startLimitIntervalSec = 0;
-    serviceConfig = {
-      Restart = "always";
-      RestartSec = "10s";
-      LimitNOFILE = 4096;
-      # Avoid flooding (and rotating too quicky) default journal with debug logs (that can't be disabled):
-      # cardano-explorer-api logs: journalctl --namespace legacy
-      LogNamespace = "legacy";
-    };
-  };
-
   services.cardano-submit-api = lib.mkIf globals.withSubmitApi {
     enable = true;
     port = 8101;
     environment = pkgs.globals.environmentConfig;
     socketPath = config.services.cardano-node.socketPath;
-    inherit cardanoNodePkgs;
+    package = cardano-submit-api;
```

Comment:
is the change necessary? it will incur a nix eval penalty (cardano-node pkgs imported twice).